apiv2/VERSION

```diff
@@ -1 +1 @@
-0.3.1
+2.0.0-dev
```

Comment:
Fixes error when importing apiv2
`go.mod:19:2: require github.com/open-edge-platform/infra-core/apiv2/v2: version "v0.3.1" invalid: should be v2, not v0`